### ISSUE TITLE
The button covers the code on mobile

### ISSUE THREAD

**Issue by NomarCub** [superDuperCyberTechno/obsidian-code-copy#2]
See the [issue](https://github.com/jdbrice/obsidian-code-block-copy/issues/11) and solution in the original repo.

**Comment by superDuperCyberTechno** [comment]
No problem. Fixing ASAP. I can't really test this on mobile so providing a solution right in the issue was awesome. Thanks.

**Comment by superDuperCyberTechno** [comment]
1.0.1 is out with fixes.

**Comment by NomarCub** [comment]
It's still not right in 1.0.1. I don't know why, but the CSS I linked has to be followed more strictly.

I pasted this at the end of the plugin's styles.css, and it works.

```css
.has-code-copy-button .code-copy-button{
    width: auto;
    margin-top: 0;
}
```

How it looks now:
![Screenshot_20210825-103856](https://user-images.githubusercontent.com/5298006/130758494-0614b3e4-3fc7-4e98-a21e-ebbbfa74f1d2.jpg)

And with strict CSS:
![Screenshot_20210825-103750](https://user-images.githubusercontent.com/5298006/130758556-f746f59e-b184-4b5c-b166-ceaa20a186e7.jpg)
